parlai/agents/hugging_face/gpt2.py

```diff
@@ -237,6 +240,12 @@ def add_cmdline_args(
         cls, parser: ParlaiParser, partial_opt: Optional[Opt] = None
     ) -> ParlaiParser:
         agent = parser.add_argument_group("Gpt2 Args")
+        agent.add_argument(
+            "--model_name",
```

Comment:
```suggestion
            "--model-name",
```

Reply:
Agree, I like this argument name more. I used --model_name because this agent uses already the argument "--gpt2_size".
If we change it to "--model-name", we should do the same in other place too, isn't it?